### DISCUSSION TITLE
#50 stuck on 99.9%

### DISCUSSION THREAD

**Discussion by Reyzartz** [cssbattle/forum#87] [Q&A] [answered]
can someone help me with this I'm stuck on 99.9% and I am unable to find the difference
![FireShot Capture 351 - Target #50 (Use Hand Sanitizer) - CSSBattle - cssbattle dev](https://user-images.githubusercontent.com/47865866/130322711-03503d00-de7a-4591-b74a-6f2bcca807f1.png)

here is my code
```
<a><b><s><i><p>
<style>
  a,b,s,i,p{
    position:fixed;
    padding:70 50;
    background:#F3AC3C;
    border-radius:22q;
    margin:102 142;
    box-shadow:inset 0 -95px 0 -13px#998235,0+0+0+3in#1A4341
  }
  b,i,p{
    padding:25;
    background:#F3AC3C;
    border-radius:11q;
    margin:-90 -25;
    box-shadow:none
  }
  s{
    padding:25;
    margin:25 -0;
    border-radius:1in;
    box-shadow:-50px 0 #998235,65px -25px 0 -15px #998235,65px -55px 0 -15px #998235
  }
  i{
    padding:10 75;
    margin:-115 -75;
  }
  p{
    padding:19 10;
    margin:0 -35;
    border-radius:15q;
    box-shadow:90px -8q #F3AC3C
  }
<style>

```

**Comment by SelenIT** [comment, marked as answer]
Apparently after the latest backend update `box-shadow: inset` doesn't work well with rounded corners for many targets, including this one. My shortest solution also used it but doesn't work anymore. For your solution, replacing the inner shadow with bottom border seems to fix the issue:
```
  a,b,s,i,p{
    ...
    padding:27 50; /* to compensate the changed inner height */
    ...
    box-shadow:0+0+0+3in#1A4341
  }
  a {
    border-bottom: solid 86px#998235
  }
  b,i,p{
    ...
    margin:-47 -25;
    ...
  }
```
